package.json

```
@@ -22,19 +22,27 @@
   "scripts": {
     "commit": "cz",
     "import": "node ./scripts/import.js",
-    "build": "node ./scripts/build.js",
+    "build": "node ./scripts/build.js && node ./scripts/i18n.js",
```

Maybe it is better to reuse the already created script? In that way it is easier to update in only one place if the script will be changed. Wdyt?
```suggestion
    "build": "node ./scripts/build.js && pnpm i18n,
```

---

Yes good idea!